compiler/src/config/common.rs

```diff
@@ -0,0 +1,14 @@
+use serde::{Serialize, Deserialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub(crate) struct Common {
+    pub log_level: u32,
```

Comment:
should this be public?

Reply:
Thanks, struct "Common" and its members should be pub(crate) - it is not a public interface